apps/aecore/src/aec_metrics_lib.erl

```diff
@@ -0,0 +1,199 @@
+-module(aec_metrics_lib).
+
+-export([find_entries_cmd/1,
+         get_values_cmd/1]).
+
+find_entries_cmd(Arg) ->
+    find_entries_cmd(Arg, enabled).
+
+find_entries_cmd(Arg, Status) ->
+    lists:map(
+      fun(A) ->
+              #{dps := DPs} = Map = type_status_and_dps(A, Status),
+              {find_entries_1(Map), DPs}
+      end, Arg).
+
+get_values_cmd(Arg) ->
+    Res = find_entries_cmd(Arg),
+    [[{Name, Type, Status, do_get_value(Name, Status, DPs)}
+      || {Name,Type,Status} <- Found] || {Found, DPs} <- Res].
+
+do_get_value(_, disabled, _) -> [];
+do_get_value(Name, _, DPs) ->
+    case exometer:get_value(Name, DPs) of
+        {ok, Value}    -> Value;
+        {error, _} = E -> E
+    end.
+
+type_status_and_dps(S, Status0) ->
+    Parts = re:split(S, "/"),
+    lists:foldl(
+      fun(P, Acc) ->
+              parse_part(part_type(P), P, Acc)
+      end, #{name   => [],
+             type   => '_',
+             status => Status0,
+             dps    => default}, Parts).
+
+part_type(<<"type=", _/binary>> ) -> type;
```

Comment:
Would be nice to somehere (not at the cost of having jsx) have a function that parses binary strings into maps.